autocommit.js

```diff
@@ -33,8 +34,9 @@ try {
   process.exit(1);
 }
 
-if (!config.openAiKey) {
-  console.error("Please set OPENAI_API_KEY");
+if (!config.openAiKey && !config.azureOpenAIApiKey
+    && !process.env.OPENAI_API_KEY && !process.env.AZURE_OPENAI_API_KEY) {
```

Comment:
this env check is redundant

Reply:
I agree that. This was mixed with the code in the middle of creation. I will remove the env check after my last two commits have done disabled with.

Reply:
@shanginn Removed redundant API environment variable check
on ba9de646dcb2b551bff616237c331ba1c4a0de7c